README.md

```diff
@@ -1,6 +1,6 @@
 # Tech Companies in Portugal :portugal:
 
-This is a opinionated list of some technology companies operating in Portugal.  
+This is a opinionated list of some technology companies operating in Portugal.
```

Comment:
Please do not remove the trailing whitespace.  
They will create a line break.

Reply:
@jmagalhaes-xing I think you might have forgot this comment.

Reply:
Fixed again. I might have removed it again on the merges